website/moneybirdsynchronization/management/commands/statementsync.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand
+import datetime
+from moneybirdsynchronization.services import MoneybirdAPIService
+from payments.models import Payment
+from moneybirdsynchronization.administration import HttpsAdministration
+from thaliawebsite import settings
+
+class Command(BaseCommand):
+    """This command needs to be executed every day to sync the bookkeeping with MoneyBird."""
+
+    def handle(self, *args, **options):
+
+        payment_identifiers = {
```

Comment:
what are payment identifiers? it are names for financial accounts I think, right (so I think it would be better to name the variable something like `payment_type_to_financial_account_mapping`)